susemanager-utils/susemanager-sls/src/grains/mgr_server.py

```diff
@@ -33,7 +34,7 @@ def server_grains():
     """ returns if this minion is a Uyuni/SUSE Manager Server and
         if it has a reporting database configured.
     """
```

Comment:
please also update the method documentation :)

Reply:
Thanks, done.